network-api/networkapi/templates/pages/buyersguide/base.html

```diff
@@ -78,22 +78,26 @@
   {% include "fragments/buyersguide/primary_nav.html" %}
   </div>
 
-  {% get_bg_home_page as home_page %}
-
-  {% block hero %}{% endblock %}
+  <div class="underneath-screen-overlay">
```

Comment:
Can you add some context around what this class is doing?

Reply:
old class, cant find any reference so removing